src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -268,7 +268,7 @@ export class BaseEditCourseForm extends React.Component {
 
     const {
       data: {
-        results: allResults,
+        results: allResults = [],
```

Comment:
if the collaborators are not fetched, how will the retry occur if data is set to empty list here?

Reply:
Its just setting a default. Once the collaborators have been fetched, the state will update which will update the collaborator options with a "non-undefined" value for `allResults` and then that will be used by the code.

[PS] Note that the collaborator info is in collaboratorOptions, and this is just pulling information out of that object.